release-notes-3.1.0.md

```diff
@@ -2,6 +2,7 @@
 #### Version Number
 - [CAF-2277](https://jira.autonomy.com/browse/CAF-2277): Exposed a new https endpoint for audit web service web calls (default 8443).
 - [CAF-2693](https://jira.autonomy.com/browse/CAF-2693): Added support for Elasticsearch 'keyword' and 'text' string data types.
+- [CAF-2846](https://jira.autonomy.com/browse/CAF-2846): Added support for configuring the Audit Web Service logging levels through an environment variable
```

Comment:
Adam I think this should be changed to Audit Service and not specific to the Web service as you have added this to the client side code as well if I recall?